From 480fb9811cf668fd07ebeb7c0f8c94481fdb81a9 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Tue, 23 Aug 2022 19:06:54 +0530
Subject: [PATCH] resolve conflicts 0031

0031-Use-lts2020-yocto-sriov-for-source-and-config-patch.patch

Tracked-On:
Signed-off-by: vdanix <vishwanathx.dani@intel.com>
---
 caas/AndroidBoard.mk | 4 ++--
 caas/mixins.spec     | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/caas/AndroidBoard.mk b/caas/AndroidBoard.mk
index 30076f3..a4bd6dd 100644
--- a/caas/AndroidBoard.mk
+++ b/caas/AndroidBoard.mk
@@ -281,8 +281,8 @@ ifeq ($(BASE_CHROMIUM_KERNEL), true)
   LOCAL_KERNEL_SRC := kernel/lts2019-chromium
   KERNEL_CONFIG_PATH := $(TARGET_DEVICE_DIR)/config-lts/lts2019-chromium
 else ifeq ($(BASE_LTS2020_YOCTO_KERNEL), true)
-  LOCAL_KERNEL_SRC := kernel/lts2020-yocto
-  KERNEL_CONFIG_PATH := $(TARGET_DEVICE_DIR)/config-lts/lts2020-yocto
+  LOCAL_KERNEL_SRC := kernel/lts2020-yocto-sriov
+  KERNEL_CONFIG_PATH := $(TARGET_DEVICE_DIR)/config-lts/lts2020-yocto-sriov
 else ifeq ($(BASE_LTS2020_CHROMIUM_KERNEL), true)
   LOCAL_KERNEL_SRC := kernel/lts2020-chromium
   KERNEL_CONFIG_PATH := $(TARGET_DEVICE_DIR)/config-lts/lts2020-chromium
diff --git a/caas/mixins.spec b/caas/mixins.spec
index f76ea30..50681b1 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -7,7 +7,7 @@ mixinsrel: false
 product.mk: device.mk
 
 [groups]
-kernel: gmin64(useprebuilt=false,src_path=kernel/lts2020-chromium, loglevel=7, interactive_governor=false, relative_sleepstates=false, modules_in_bootimg=false, external_modules=,debug_modules=, use_bcmdhd=false, use_iwlwifi=false, extmod_platform=bxt, iwl_defconfig=, cfg_path=config-lts/lts2020-chromium, more_modules=true, chromium_src_path=kernel/lts2019-chromium, chromium_cfg_path=config-lts/lts2019-chromium, lts2020_yocto_src_path=kernel/lts2020-yocto, lts2020_yocto_cfg_path=config-lts/lts2020-yocto, lts2020_chromium_src_path=kernel/lts2020-chromium, lts2020_chromium_cfg_path=config-lts/lts2020-chromium)
+kernel: gmin64(useprebuilt=false,src_path=kernel/lts2020-chromium, loglevel=7, interactive_governor=false, relative_sleepstates=false, modules_in_bootimg=false, external_modules=,debug_modules=, use_bcmdhd=false, use_iwlwifi=false, extmod_platform=bxt, iwl_defconfig=, cfg_path=config-lts/lts2020-chromium, more_modules=true, chromium_src_path=kernel/lts2019-chromium, chromium_cfg_path=config-lts/lts2019-chromium, lts2020_yocto_src_path=kernel/lts2020-yocto-sriov, lts2020_yocto_cfg_path=config-lts/lts2020-yocto-sriov, lts2020_chromium_src_path=kernel/lts2020-chromium, lts2020_chromium_cfg_path=config-lts/lts2020-chromium)
 disk-bus: auto
 boot-arch: project-celadon(uefi_arch=x86_64,fastboot=efi,ignore_rsci=true,disable_watchdog=true,watchdog_parameters=10 30,verity_warning=false,txe_bind_root_of_trust=false,bootloader_block_size=4096,verity_mode=false,disk_encryption=false,file_encryption=true,metadata_encryption=true,fsverity=true,target=caas,ignore_not_applicable_reset=true,self_usb_device_mode_protocol=true,usb_storage=true,live_boot=true,userdata_checkpoint=true)
 sepolicy: enforcing
-- 
2.37.2

